packages/dialect-react-ui/components/Chat/screens/Main/pages/CreateThreadPage/CreateThread.tsx

```diff
@@ -359,7 +394,14 @@ export default function CreateThread({
           onChange={(e) => onAddressChange(e.target.value)}
         />
         {!isTyping
-          ? showAddressResult(validAddress, address, cardinalAddress, snsAddress)
+          ? showAddressResult(
+              validAddress,
```

Comment:
That's a lot of arguments :D

Might want to rethink the signature of that function later. No need to do anything now.

Reply:
Yes, let's refactor this function I think some of the logic can be simplified in there.